app/main.py

```diff
@@ -1 +1,29 @@
-# write your code here
+from app.players.player import Player
+from app.players.dwarves.dwarf import Dwarf
+from app.players.dwarves.dwarf_blacksmith import DwarfBlacksmith
+from app.players.dwarves.dwarf_warrior import DwarfWarrior
+from app.players.elves.druid import Druid
+from app.players.elves.elf import Elf
+from app.players.elves.elf_ranger import ElfRanger
+
+
+def calculate_team_total_rating(players: list) -> int:
+    return sum([player.get_rating() for player in players])
+
+
+def elves_concert(elves: list) -> None:
+    [elf.play_elf_song() for elf in elves]
+
+
+def feast_of_the_dwarves(dwarves: list) -> None:
+    [dwarf.eat_favourite_dish() for dwarf in dwarves]
```

Comment:
Add more detailed annotation for args and use standard loop. No need to use list comprehension when you don't need a result